tez-plugins/tez-aux-services/pom.xml

```diff
@@ -241,15 +242,6 @@
                   <pattern>io.netty</pattern>
                   <shadedPattern>org.apache.tez.shaded.$0</shadedPattern>
                 </relocation>
-                <relocation>
-                  <pattern>javax</pattern>
-                  <shadedPattern>org.apache.tez.shaded.$0</shadedPattern>
-                  <excludes>
-                    <exclude>javax.crypto.*</exclude>
-                    <exclude>javax.security.**</exclude>
-                    <exclude>javax.net.**</exclude>
-                  </excludes>
-                </relocation>
```

Comment:
I can see the following jars are currently relocated.
- `org/apache/tez/shaded/javax/annotation/`
- `org/apache/tez/shaded/javax/xml/bind/`

I guess they come from Hadoop. So, "As javax is part of the jdk:" in the ticket might not be correct. Personally, I don't have any objections to exclude javax, though

```
+- org.apache.hadoop:hadoop-yarn-api:jar:3.3.6:provided
|  +- javax.xml.bind:jaxb-api:jar:2.2.11:compile
```